ethcore/src/miner/miner.rs

```diff
@@ -1146,37 +1162,44 @@ mod tests {
 		assert!(miner.submit_seal(hash, vec![]).is_ok());
 	}
 
+	fn  default_miner_opts() -> MinerOptions {
+		MinerOptions {
+			force_sealing: false,
+			reseal_on_external_tx: false,
+			reseal_on_own_tx: true,
+			reseal_on_uncle: false,
+			reseal_min_period: Duration::from_secs(5),
+			reseal_max_period: Duration::from_secs(120),
+			pending_set: PendingSet::AlwaysSealing,
+			work_queue_size: 5,
+			enable_resubmission: true,
+			infinite_pending_block: false,
+			tx_queue_penalization: Penalization::Disabled,
+			tx_queue_strategy: PrioritizationStrategy::GasPriceOnly,
+			tx_queue_allow_unknown_local: true,
+			refuse_service_transactions: false,
+			pool_limits: Default::default(),
+			pool_verification_options: pool::verifier::Options {
+				minimal_gas_price: 0.into(),
+				block_gas_limit: U256::max_value(),
+				tx_gas_limit: U256::max_value(),
+			},
+		}
+	}
+
 	fn miner() -> Miner {
 		Miner::new(
-			MinerOptions {
-				force_sealing: false,
-				reseal_on_external_tx: false,
-				reseal_on_own_tx: true,
-				reseal_on_uncle: false,
-				reseal_min_period: Duration::from_secs(5),
-				reseal_max_period: Duration::from_secs(120),
-				pending_set: PendingSet::AlwaysSealing,
-				work_queue_size: 5,
-				enable_resubmission: true,
-				infinite_pending_block: false,
-				tx_queue_penalization: Penalization::Disabled,
-				tx_queue_strategy: PrioritizationStrategy::GasPriceOnly,
-				refuse_service_transactions: false,
-				pool_limits: Default::default(),
-				pool_verification_options: pool::verifier::Options {
-					minimal_gas_price: 0.into(),
-					block_gas_limit: U256::max_value(),
-					tx_gas_limit: U256::max_value(),
-				},
-			},
+			default_miner_opts(),
```

Comment:
Why having an extra `default_miner_opts`? I think this may be confusing as `MinerOptions` already implements `Default` trait.

Reply:
Yup, it is confusing. The config returned is the default used in testing (which is not Default:: default()).

probs better to extract config from the miner produced by miner()

Reply:
Alternative is to rename it to something like `std_test_miner_opts`